account_invoice_triple_discount/models/account_move.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    discount1 = fields.Float("Discount 1 (%)", digits="Discount")
+    discount2 = fields.Float("Discount 2 (%)", digits="Discount")
+    discount3 = fields.Float("Discount 3 (%)", digits="Discount")
+
+    @api.onchange("discount1", "discount2", "discount3")
+    def _onchange_triple_discount(self):
```

Comment:
should not be a compute function ? otherwise, a self.env["account.move.line"].create({"discount1": 10, "discount2": 20}) will not raise the compute of discount. Don't you think ?

Reply:
Thank you, this is a good comment.
Fixed.